ui/component/or-image-map/src/index.ts

```diff
@@ -0,0 +1,359 @@
+import {debounce} from "lodash";
+import { getAssetDescriptorIconTemplate } from "@openremote/or-icon";
+
+//language=css
+const styling = css`
+    :host {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+    }
+
+    .chart-wrapper {
+        position: relative;
+        /*flex: 1;*/
+        width: 100%;
+    }
+
+    #chart {
+        width: 100%;
+    }
+
+    .chart-description {
+        margin: 0 20px;
+        font-size: 16px;
+        z-index: 3;
+    }
+    .mainvalue-wrapper {
+        width: 100%;
+        display: flex;
+        flex: 0 0 60px;
+        align-items: center;
+        justify-content: center;
+    }
+    .main-number {
+        color: var(--internal-or-asset-viewer-title-text-color);
+        font-size: 42px;
+    }
+
+    .main-number-icon {
+        font-size: 24px;
+        margin-right: 10px;
+        display: flex;
+    }
+
+    .main-number-unit {
+        font-size: 42px;
+        color: var(--internal-or-asset-viewer-title-text-color);
+        font-weight: 200;
+        margin-left: 5px;
+    }
+
+    .main-number.xs, .main-number-unit.xs {
+        font-size: 18px;
+    }
+    .main-number.s, .main-number-unit.s {
+        font-size: 24px;
+    }
+    .main-number.m, .main-number-unit.m {
+        font-size: 30px;
+    }
+    .main-number.l, .main-number-unit.l {
+        font-size: 36px;
+    }
+    .main-number.xl, .main-number-unit.xl {
+        font-size: 42px;
+    }
+`
+// export interface OrGaugeConfig {
+//     attributeRef?: AttributeRef;
+//     thresholds?: [number, string][]
+//     options?: GaugeOptions;
+// }
+
+export interface OrImageConfig {
+    attributeRef?: AttributeRef;
+    imageName?: string;
+    imageUploaded?: boolean;
+    imagePath?: string;
+}
+
+@customElement("or-image")
+export class OrImageMap extends LitElement {
+    public static styles = styling;
+
+    @query("#image")
+    private _imageElem!: HTMLCanvasElement;
+
+    private resizeObserver?: ResizeObserver;
+
+    @property()
+    public readonly config?: OrImageConfig;
+
+    constructor() {
+        //initialise custom component
+        super();
+        if(!this.config) {
+            this.config = this.getDefaultConfig();
+        }
+        this.updateComplete.then(() => {
+            this.resizeObserver = new ResizeObserver(debounce((entries: ResizeObserverEntry[]) => {
+                // this.setupGauge(); // recreate gauge since the library is not 100% responsive.
+                const gaugeSize = entries[0].devicePixelContentBoxSize[0].blockSize; // since gauge elem width == value elem width
+                // this.setLabelSize(gaugeSize);
+            }, 200))
+            this.resizeObserver.observe(this.shadowRoot!.getElementById('image')!);
+        })
+    }
+
+    render() {
+        return html
+        `<div id="container">
+            <img class="image" src="${this.config?.imagePath}" alt="${this.config?.imageName}"></img>
```

Comment:
You could do some tests to check what happens when `imagePath` or `imageName` are undefined.
I could imagine a fallback image would be nice 😄